$NetBSD: patch-as,v 1.1 2006/10/26 13:54:25 rillig Exp $

sunpro is a little picky about overloaded functions, for good reason.

--- wxxt/src/DeviceContexts/WindowDC.cc.orig	2004-12-19 09:38:26.000000000 +0100
+++ wxxt/src/DeviceContexts/WindowDC.cc	2006-10-26 15:45:24.638175276 +0200
@@ -1935,7 +1935,7 @@ void wxWindowDC::DrawText(char *text, fl
 	  fontinfo = (XFontStruct *)current_font->GetInternalFont(scale_x, scale_y, angle);
 	  XSetFont(DPY, TEXT_GC, fontinfo->fid);
 
-	  quadrant = fmod(angle, 2 * pie);
+	  quadrant = fmod((double)angle, 2 * pie);
 	  if (quadrant < 0)
 	    quadrant += (2 * pie);
 	  
